templates/content_hub_registration_box.mustache

```diff
@@ -2,7 +2,7 @@
     <div class="form-label col-sm-3 text-sm-right"></div>
     <div class="form-setting col-sm-9">
         {{^name}}
-            {{# str }} contenthub:register, mod_hvp, {{ registrationurl }} {{/str}}
+            {{#register}} contenthub:register, mod_hvp, {{ registrationurl }} {{/register}}
```

Comment:
Instead of this being a # block which is for iteration in mustache, kill this entire line and just make it <p> {{register}} </p> and thats all you need to do